demo/app/home/home.component.html

```diff
@@ -13,13 +13,14 @@
 </style>
 
 <mat-card>
-    <q-grid id="q-grid-demo"            
+    <q-grid id="q-grid-demo"
             q-grid-autofocus
             [columns]="columns"
             [rows]="rows"
             [filterFetch]="filterFetch"
             [sortBy]="['+name.last', '-name.first']"
-            editMode="cell"
+            [selectionMode]="'range'"
```

Comment:
Edit mode batch

Reply:
What will be is we remove selectionMode from here?

Reply:
then marker will be invisible